rootfs/deis/settings.py

```diff
@@ -284,8 +284,8 @@
 TEST_RUNNER = 'api.tests.SilentDjangoTestSuiteRunner'
 
 # etcd settings
-ETCD_HOST = os.environ.get('DEIS_ETCD_1_SERVICE_HOST', '127.0.0.1')
-ETCD_PORT = os.environ.get('DEIS_ETCD_1_SERVICE_PORT_CLIENT', 4001)
+ETCD_HOST = os.environ.get('ETCD_SERVICE_HOST', '127.0.0.1')
```

Comment:
I've been testing with deis/etcd, and `DEIS_ETCD_1_SERVICE_HOST` and friends are the environment variables it makes available, so I don't think this will work without some changes over there.